image/create_private_image.sh

```diff
@@ -0,0 +1,176 @@
+#!/usr/bin/env bash
```

Comment:
Also unrelated to CUPPA 2.1.x. Did we somehow lose history of this file?

Reply:
All of these came in from 5.34-hotfix. As mentioned in the commit messages I retained the changes made to the imaging scripts but the rest has come from there.